src/webgpu/api/operation/command_buffer/render/state_tracking.spec.ts

```diff
@@ -179,3 +184,98 @@ g.test('set_index_buffer_without_changing_buffer')
       );
     }
   });
+
+g.test('set_vertex_buffer_without_changing_buffer')
+  .desc(
+    `
+  Test that setting vertex buffer states (offset, size) multiple times in different orders still
+  keeps the correctness of each draw call.
```

Comment:
nit: Describe in a little more detail, like "Tries several different sequences of setVertexBuffer+draw commands, each of which draws vertices in all 4 output pixels, and check they were drawn correctly."